tests/test_check_db_migrated.py

```diff
@@ -0,0 +1,93 @@
+"""Test check_db_migrated function."""
+
+from pyramid.config import Configurator
+from pyramid_deferred_sqla import check_db_migrated
+from unittest import mock
+
+import base64
+import pytest
+import urllib
+
+
+def test_SKIP_CHECK_DB_MIGRATED() -> None:
+    """Support skipping the check with a config flag."""
+    config = Configurator()
+    config.registry.settings["SKIP_CHECK_DB_MIGRATED"] = "true"
+
+    assert check_db_migrated(config) is None
```

Comment:
This test is a bit confusing. Since `check_db_migrated` always just `return`s and no path returns any non-None value.
Later I realized that other "path" is `sys.exit` in which case,  (a different) test checks whether `sys.exit` was called.

I would have "exited" from the caller if certain `check_...` function returned equivalent of "check failed" instead of "exiting" from inside a function.